torch/csrc/cudnn/cudnn-wrapper.h

```diff
@@ -0,0 +1,9 @@
+#ifndef THP_CUDNN_CUDNN_WRAPPER_INC
+#define THP_CUDNN_CUDNN_WRAPPER_INC
+
+#include <cudnn.h>
+#if CUDNN_MAJOR < 6
+#error CuDNN v5 and lower not supported. Please update or build without it.
```

Comment:
"Please update or build without it" -> "Please install CuDNN v6 or greater. You can get the latest version of CuDNN from https://developer.nvidia.com/cudnn"

Reply:
actually, better:
"CuDNN v5 and lower not supported...." -> "CuDNN v" #CUDNN_MAJOR " found, but need atleast CuDNN v6. Get the latest version of CuDNN from https://developer.nvidia.com/cudnn"

Reply:
#error doesn't support using macros after it (it just dumps whatever is after it literally) but after some preprocessor hackery I think I have something that works like this...